README.md

```diff
@@ -266,7 +266,7 @@ class MyFacebookAuthenticator extends SocialAuthenticator
     public function supports(Request $request)
     {
         // continue ONLY if the URL matches the check URL
-        return $request->getPathInfo() == '/connect/facebook/check';
+        return $request->getPathInfo() == $this->router->getRouteCollection()->get('connect_facebook_check')->getPath();
```

Comment:
Why don't just `$this->router->generate('connect_facebook_check')`?

Reply:
would add a host when the route has a host defined or?

Reply:
It shouldn't add a host, but I'm actually not sure if you have the host defined - good point :).

But, there another option anyways, where we look specifically at the *route* name:

```php
return $request->attributes->get('_route') == 'connect_facebook_check';
```

Do you guys like that better?

Reply:
Looks good for me did update the PR